twarc_hashtags.py

```diff
@@ -62,20 +59,43 @@ def load(infile, outfile, db):
 
             data = json.loads(line)
             for tweet in ensure_flattened(data):
+                # Process Retweets:
+                if "referenced_tweets" in tweet:
+                    rts = [
```

Comment:
Maybe we should consider hashtags in quoted tweets too?

Reply:
I thought about that but i was unsure how to handle it. Maybe it should have 1 switch for all? `--include-referenced-tweets` or ``--count-referenced-tweets` or add ones for each type: `--count-replies` and `--count-quotes` etc?